src/app/components/import-annotations/edit/edit.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from "@angular/core";
+import { List } from "immutable";
+import {
+  AudioEventImportService,
+  audioEventImportResolvers,
+} from "@baw-api/audio-event-import/audio-event-import.service";
+import { ActivatedRoute, Router } from "@angular/router";
+import { ToastrService } from "ngx-toastr";
+import { AudioEventImport } from "@models/AudioEventImport";
+import {
+  FormTemplate,
+  defaultSuccessMsg,
+} from "@helpers/formTemplate/formTemplate";
+import { annotationMenuActions } from "../details/details.component";
+import schema from "../audio-event-import.schema.json";
+import { annotationsImportCategory, editAnnotationImportMenuItem } from "../import-annotations.menu";
+
+const audioEventImportKey = "audioEventImport";
+
+@Component({
+  selector: "baw-edit-annotation",
+  template: `
+    <baw-form
+      *ngIf="!failure"
+      [title]="'Edit ' + model.name"
+      [model]="model"
+      [fields]="fields"
+      [submitLoading]="loading"
+      submitLabel="Submit"
+      (onSubmit)="submit($event)"
+    ></baw-form>
+  `,
+})
+class EditAnnotationsComponent
+  extends FormTemplate<AudioEventImport>
+  implements OnInit
+{
+  public constructor(
+    private api: AudioEventImportService,
+    route: ActivatedRoute,
+    notifications: ToastrService,
+    router: Router
+  ) {
+    super(notifications, route, router, {
+      getModel: (models) => models[audioEventImportKey] as AudioEventImport,
+      successMsg: (model) => defaultSuccessMsg("updated", model.name),
```

Comment:
Easy to fix. Just letting you know: this will change a large number of files as this is defined in our [formTemplate.ts](https://github.com/QutEcoacoustics/workbench-client/blob/master/src/app/helpers/formTemplate/formTemplate.ts#L20-L32)

![image](https://github.com/QutEcoacoustics/workbench-client/assets/33742269/9c3435b2-529d-464b-a31c-c54c6979cc10)

Reply:
separate PR then